### PR TITLE
New import account screens

### PR DESCRIPTION
**Pull request by gguij004**
### 🗃 Github Issue Or Explanation for this PR. (What is it supposed to do and Why is needed)

No issue

### ✅ Checklist

- [x] Github issue details are up to date for people to QA.
- [x] I have tested all my changes.

### 🕵️‍♂️ Notes for Code Reviewer

- Removed import word screen.
- Made import screen stateless.
- Added new create nickname screen

### 🙈 Screenshots

image

image

image

### 👯‍♀️ Paired with

 "nobody"

### CODE REVIEW

File: lib/screens/authentication/import_key/components/import_key_accounts_widget.dart [hunk 1]
```diff
@@ -1,74 +1,74 @@
-import 'package:flutter/material.dart';
```
Comment (gguijarro-c-chwy) [inline lib/screens/authentication/import_key/components/import_key_accounts_widget.dart:1]: delete the file?

File: lib/screens/authentication/create_nickname/create_nickname_screen.dart [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                      ),
+                      const Spacer(),
+                      BlocBuilder<CreateNicknameBloc, CreateNicknameState>(
+                        buildWhen: (previous, current) {
+                          return previous.isButtonLoading != current.isButtonLoading;
+                        },
+                        builder: (context, state) {
+                          return FlatButtonLong(
+                            isLoading: state.isButtonLoading,
+                            title: "Next (2/2 )",
```
[the remark this answers is not in the document]
Comment (gguij004) [reply lib/screens/authentication/create_nickname/create_nickname_screen.dart]: ty, fix

File: lib/screens/authentication/import_key/components/import_key_accounts_widget.dart (hunk 1 above)
Comment (gguij004) [reply lib/screens/authentication/import_key/components/import_key_accounts_widget.dart:1]: yes you are right, deleted

**Review by gguijarro-c-chwy — APPROVED** [review]

**Review by n13 — APPROVED** [review]

**Comment by n13** [comment]
merged and updated my pr

**Pull request merged**